src/PortingAssistant.Client.Common/Model/ProjectAnalysisResult.cs

```diff
@@ -20,6 +20,7 @@ public class ProjectAnalysisResult : ProjectDetails, IDisposable
         public ExternalReferences ExternalReferences { get; set; }
         public ProjectCompatibilityResult ProjectCompatibilityResult { get; set; }
 
+        public bool HasError { get; set; } //GetCompatibilityResponse HasError 
```

Comment:
If this only signals errors from compatibility checking, can we rename it to `HasCompatibilityCheckingError`? Also, is it possible to save these errors to a property as well? Something like `CompatibilityCheckingErrors`.

Reply:
sure.  Updated.